config/src/kura.rs

```diff
@@ -23,6 +23,8 @@ pub struct Configuration {
     pub blocks_per_storage_file: NonZeroU64,
     /// Default buffer capacity of actor's MPSC channel.
     pub actor_channel_capacity: u32,
+    /// Store new blocks as plain text.
+    pub new_blocks_are_plain_text: bool,
```

Comment:
This is a bit deceiving. The Kura struct is mutable only during initialization and after that is naked inside an Arc. So it can't change while iroha is running. As for it being able to change between runs on the same block store, that I think is important. You may run in binary mode, but when you have an issue want to switch it to plain text.